.travis.yml

```diff
@@ -18,12 +25,12 @@ deploy:
   on:
     tags: true
     repo: openzipkin/zipkin-ruby
-    condition: "$TRAVIS_RUBY_VERSION == 2.6"
+    condition: "$TRAVIS_RUBY_VERSION == 2.7"
```

Comment:
Travis is trying to deploy on both gemfiles...
https://travis-ci.org/github/openzipkin/zipkin-ruby/jobs/675596484